enclave_wrangler/objects.py

```diff
@@ -67,21 +64,10 @@ def obj_types(self) -> List[Dict]:
         # types = sorted([x['apiName'] for x in response_json])
         return response_json['data']
 
-    # TODO: Implement this func to get all concept sets
-    #  - handle pagination. I think get 1k at a time
-    # TODO: after: for each set, get all concepts.
-    # TODO: Look at Siggie's frontend/ code to see how OMOPConceptSet etc are being called/used
-    # TODO: Need to find the right object_type, then write a wrapper func around this to get concept sets
-    #  - To Try: CodeSystemConceptSetVersionExpressionItem, OMOPConcept, OMOPConceptSet, OMOPConceptSetContainer,
-    #    OmopConceptSetVersionItem
-    def objects(self, object_type) -> pd.DataFrame:
-        """Get objects
-        Docs: https://www.palantir.com/docs/foundry/api/ontology-resources/objects/list-objects/
-        https://www.palantir.com/docs/foundry/api/ontology-resources/objects/object-basics/"""
-        # Request
+    def _handle_paginated_request(self, first_page_url: str) -> (List[Dict], Response):
```

Comment:
I figured it would be quite handy to re-use this pagination-handling function, so I refactored it from within `EnclaveClient.objects()` to `EnclaveClient._handle_paginated_request()`